.vscode/mcp.json

```diff
@@ -0,0 +1,14 @@
+{
+  "servers": {
+    "pinner-mcp": {
+      "type": "stdio",
+      "command": "docker",
+      "args": [
+        "run",
+        "--rm",
+        "-i",
+        "ghcr.io/safedep/pinner-mcp:latest"
```

Comment:
This config is used only for this project itself right?

Local dev requires testing out unpublished versions